easybuild/easyconfigs/q/Qt/Qt-4.8.6-intel-2015a.eb

```diff
@@ -1,19 +1,32 @@
+#easyblock = 'CMakeMake'
+
 name = 'Qt'
 version = '4.8.6'
 
 homepage = 'http://qt-project.org/'
 description = "Qt is a comprehensive cross-platform C++ application framework."
 
-toolchain = {'name': 'intel', 'version': '2015a'}
+toolchain = {'name': 'intel', 'version': '2015a'} 
 
-source_urls = [
-    'http://origin.releases.qt-project.org/qt4/source/',
-    'http://master.qt-project.org/archive/qt/%(version_major_minor)s/%(version)s/'
-]
+source_urls = ['http://download.qt-project.org/official_releases/qt/4.8/4.8.6/']
 sources = ['%(namelower)s-everywhere-opensource-src-%(version)s.tar.gz']
 
-patches = ['Qt-4.8.6_icc-qUnused.patch']
 
-dependencies = [('GLib', '2.40.0')]
+dependencies = [
+    ('GLib', '2.42.1'),
+    ('Mesa', '7.11.2', '-Python-2.7.9'),
+    ('xrandr', '1.4.3'),
+    ('libXrender','0.9.8'),
+]
+
+patches = [ 'Qt-4.8.6_icc-qUnused.patch' ]
+# patches = [ 'Qt-4.8.5-foss-2014a-OpenGL-qmake.patch2']
+
+configopts =" -confirm-license -opensource -silent -opengl -glib -webkit -developer-build -xrender"
```

Comment:
this significantly extends the Qt functionality; that's good, but it also pulls in a bunch of other dependencies...

especially Mesa is quite painful, so I'm not sure whether including it as a dependendy for Qt by default is a good idea

maybe this should be a variant with a custom versionsuffix, rather than a 'standard' build?

thoughts, @gjbex?